config/testgrids/kubernetes/k8s-infra-public-pii/config.yaml

```diff
@@ -0,0 +1,7 @@
+dashboard_groups:
+  - name: k8s-infra-public-pii
+    dashboard_names:
+      - k8s-infra-public-pii-all
+
+dashboards:
+  - name: k8s-infra-public-pii-all
```

Comment:
This should go into the sig-k8s-infra dashboard group, with a dashboard named something like sig-k8s-infra-artifacts